docs/plugins/mapper.asciidoc

```diff
@@ -10,7 +10,8 @@ The core mapper plugins are:
 
 <<mapper-attachments>>::
 
-The mapper-attachments integrates http://lucene.apache.org/tika/[Apache Tika] to provide a new field
+deprecated[3.0.0, Mapper attachments has been deprecated by <<ingest-attachment>>]: The mapper-attachments integrates
```

Comment:
Same here:

    deprecated[3.0.0,The `mapper-attachments` plugin has been replaced by the {plugins}/ingest-attachment.html[`ingest-attachment`] plugin]